odo/backends/dask.py

```diff
@@ -12,12 +12,14 @@
 from dask.compatibility import long
 import dask.dataframe as dd
 
-from odo import append, chunks, convert, discover, TextFile
+from odo import append, discover, convert
+from odo.chunks import chunks
+from odo.backends.text import TextFile
 from ..utils import filter_kwargs
 
 
 @discover.register(Array)
-def discover_dask_array(a, **kwargs):
+def discover_dask_array(a, **_):
```

Comment:
this change is not needed, this pattern is used all over and it is expected that `**kwargs` may or may not be used.